content/en/blog/_posts/YYYY-MM-DD-streaming-list-responses/YYYY-MM-DD-streaming-list-responses.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+title: "Streaming List responses for K8s 1.33"
+date: YYYY-MM-DD
```

Comment:
We can use the `draft: true` front matter, which allows us to control the publication timing at a later date (such contents won't make it to the official blog). We, the Release Comms team, can handle the actual publication strategy with that!

```suggestion
date: YYYY-MM-DD
draft: true
```